var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -135,9 +135,6 @@ class Llvm(CMakePackage, CudaPackage):
     depends_on("python", when="@5:+python")
     depends_on("z3", when="@9:")
 
-    # CUDA dependency
-    depends_on("cuda", when="+cuda")
```

Comment:
I think you can also delete the line:
```python
conflicts("cuda_arch=none", when="+cuda", msg="A value for cuda_arch must be specified.")
```
later on in the file, since that's part of the CudaPackage.

Reply:
No, this conflict is not part of the CudaPackage.